### ISSUE TITLE
cygwin installation: error installing helper binaries

### ISSUE THREAD

**Issue by ideruga** [begriffs/haskell-vim-now#132]
In cygwin, while running the installation, I get this error:

```
 --- Installing helper binaries...
*** Binary installation failed with error 1. Aborting...
--- Installing git-hscope...
While constructing the BuildPlan the following exceptions were encountered:

 --  Failure when adding dependencies:
      haskell-src-exts: needed (>=1.17.0), 1.16.0.1 found (latest is 1.17.1)
    needed for package: pointful-1.0.7
--- Building Hoogle database...
```

I re-ran the line explicitly with the following result:

```
$ stack --resolver nightly-2015-12-08 install ghc-mod hdevtools hasktags codex hscope pointfree pointful hoogle stylish-haskell --verbosity warning
While constructing the BuildPlan the following exceptions were encountered:

--  Failure when adding dependencies:
      haskell-src-exts: needed (==1.17.*), 1.16.0.1 found (latest is 1.17.1)
    needed for package: pointfree-1.1.1.1

--  Failure when adding dependencies:
      haskell-src-exts: needed (>=1.17.0), 1.16.0.1 found (latest is 1.17.1)
    needed for package: pointful-1.0.7
```

I tried different snapshots and no snapshots at all and there is always this version conflict. For exemple, when I run it with dec-09 snapshot, I get this:

```
$ stack --resolver nightly-2015-12-09 install ghc-mod hdevtools hasktags codex hscope pointfree pointful hoogle stylish-haskell --verbosity warning
While constructing the BuildPlan the following exceptions were encountered:

--  Failure when adding dependencies:
      haskell-src-exts: needed (==1.16.*), 1.17.1 found (latest version available)
    needed for package: hscope-0.4.2
```

I am not really sure how to address this problem, since my stack knowledge is insufficient.

**Comment by SimpleX91** [comment]
Try setting lts resolvers. Like lts-3.20 (and lower).

**Comment by Rydgel** [comment]
Same problem on Mac.

**Comment by ideruga** [comment]
@SimpleX91: tried with lts-3.20:
```
$ stack --resolver lts-3.20 install ghc-mod hdevtools hasktags codex hscope pointfree pointful hoogle stylish-haskell --verbosity warning
While constructing the BuildPlan the following exceptions were encountered:

--  Failure when adding dependencies:
      haskell-src-exts: needed (==1.17.*), 1.16.0.1 found (latest is 1.17.1)
    needed for package: pointfree-1.1.1.1

--  Failure when adding dependencies:
      haskell-src-exts: needed (>=1.17.0), 1.16.0.1 found (latest is 1.17.1)
    needed for package: pointful-1.0.7
```

The same thing with 3.01 (and others in-between). The 2.22 version and below don't work because of the ghc version.

```
$ stack --resolver lts-3.01 install ghc-mod hdevtools hasktags codex hscope pointfree pointful hoogle stylish-haskell --verbosity warning
While constructing the BuildPlan the following exceptions were encountered:

--  Failure when adding dependencies:
      haskell-src-exts: needed (==1.17.*), 1.16.0.1 found (latest is 1.17.1)
    needed for package: pointfree-1.1.1.1

--  Failure when adding dependencies:
      haskell-src-exts: needed (>=1.17.0), 1.16.0.1 found (latest is 1.17.1)
    needed for package: pointful-1.0.7
```

**Comment by begriffs** [comment]
I'm getting the error on my mac as well now. It's because the helper library `pointful` released 1.0.7 which requires a newer `haskell-src-exts` while other helpers require an older one. To solve this particular problem I'll make a commit that pins the `pointful` version at 1.0.6. The more future-proof solution is to pin the versions of all helpers. I thought the stackage release did that, but maybe not so well.

**Comment by begriffs** [comment]
(This fixes it for me, but reopen the issue if it's not fixed for you)
